src/segments/Feedbox.tsx

```diff
@@ -11,32 +11,29 @@ import styled, { ThemeProvider, useTheme } from "styled-components";
 // Internal Compoonents
 import * as PushAPI from "@pushprotocol/restapi";
 import { NotificationItem } from "@pushprotocol/uiweb";
+import { ReactComponent as Close } from 'assets/chat/group-chat/close.svg';
```

Comment:
- height: 85% causes bottom gap in inbox and spam
- Feedbox should be renamed to Inbox
- spam should be Spam
- ScrollItem should be duplicated and adjusted instead of imported
- height 100% causes loading indicator to hide
- why is dispatch needed provided notifications are loaded internally